arktype.io/src/components/features.tsx

```diff
@@ -0,0 +1,62 @@
+/* eslint-disable max-lines-per-function */
+import { Box, Stack, SvgIcon, Typography } from "@mui/material"
+import One from "@site/static/img/features/1.svg"
+import Two from "@site/static/img/features/2.svg"
+import Three from "@site/static/img/features/3.svg"
+import Four from "@site/static/img/features/4.svg"
+import React from "react"
+import details from "../featuresConfig.json"
+
+const features = {
+    One: <One />,
+    Two: <Two />,
+    Three: <Three />,
+    Four: <Four />
+}
+
+type FeatureProps = {
+    image: JSX.Element
+    title: string
+    description: string
+}
+
+const Feature = ({ image, title, description }: FeatureProps) => (
+    <Box id="feature" flex="1 0 45%">
+        <SvgIcon
+            sx={{
+                height: 150,
+                width: "100%"
+            }}
+        >
+            {image}
+        </SvgIcon>
+        <Typography component="h3" variant="h5" id="title">
+            {title}
+        </Typography>
+        <Typography component="h3" variant="h6" id="description">
+            {description}
+        </Typography>
+    </Box>
+)
+
+const feats = details.features.map((feature, i) => (
+    <Feature
+        image={features[feature.image as keyof typeof features]}
```

Comment:
One advantage to including the object in TS as opposed to external JSON is that you have more control over how the type is inferred. This is likely a situation where it's better to ensure the original `feature.image` type is correct rather than casing it later.